src/plugin/advancedFormat/index.js

```diff
@@ -11,6 +11,11 @@ export default (o, c, d) => { // locale needed later
   // extend en locale here
   proto.format = function (formatStr) {
     const locale = this.$locale()
+
```

Comment:
if (!this.isValid()) {
      return oldFormat.bind(this)(formatStr)
    }

Reply:
thx. i've learned from this comment